cli/lib/kontena/cli/stacks/install_command.rb

```diff
@@ -19,14 +19,16 @@ class InstallCommand < Kontena::Command
 
     option '--parent-name', '[PARENT_NAME]', "Set parent stack name", hidden: true
     option '--skip-dependencies', :flag, "Do not install any stack dependencies"
+    option '--dry-run', :flag, "Simulate install"
 
     requires_current_master
     requires_current_master_token
 
     def execute
       set_env_variables(stack_name, current_grid)
 
-      install_dependencies unless skip_dependencies?
+      values_from_installed_stacks.merge!(install_dependencies) unless skip_dependencies?
```

Comment:
This would be easier to read with a named local variable. Something like:

```
unless skip_dependencies?
  values_from_dependencies = install_depenencies
  values_from_installed_stacks.merge!(values_from_dependencies)
end
```